packages/gravity-ui-web/src/ui-lib/sass/01-tools/_color-schemes.scss

```diff
@@ -0,0 +1,104 @@
+////
+/// Mixins for defining color schemes in CSS.
+///
+///
+/// @group  colors
+/// @access public
+////
+
+@function grav-color-scheme-map($name: $grav-co-scheme-default-name) {
+  @return map-get($grav-co-schemes, $name);
+}
+
+
```

Comment:
Is there a linter like stylelint available? Would be good to disable excessive whitespace lines here.

Reply:
We have stylelint, but I guess it didn't flag this. Will take a look at the config.